parachain/primitives/router/src/outbound/mod.rs

```diff
@@ -97,17 +81,12 @@ where
 			SendError::MissingArgument
 		})?;
 
-		let mut converter = XcmConverter::new(&message, &gateway_network, &gateway_address);
-		let (agent_execute_command, max_target_fee) = converter.convert().map_err(|err|{
+		let mut converter = XcmConverter::new(&message, &gateway_network);
+		let agent_execute_command = converter.convert().map_err(|err|{
 			log::error!(target: "xcm::ethereum_blob_exporter", "unroutable due to pattern matching error '{err:?}'.");
 			SendError::Unroutable
 		})?;
 
-		if max_target_fee.is_some() {
-			log::error!(target: "xcm::ethereum_blob_exporter", "unroutable due not supporting max target fee.");
-			return Err(SendError::Unroutable)
-		}
-
```

Comment:
Do we also need to check `max_target_fee` is greater than the `fee` calculated from `OutboundQueue::validate(&outbound_message)`?

Reply:
This fee is in WETH not DOT and we do not actually charge the fee, or rather we charge a 0 fee. So we do not need to check it here. `OutboundQueue::validate(&outbound_message)` returns a fee in DOT. The `reserve_transfer` extrinsic charges the end user the DOT fee and embeds it into the XCM program.